demos/tests/cases.py

```diff
@@ -413,6 +511,34 @@ def single_option_cases(key, *args):
                     ModelArg('deeplabv3'))),
         ],
     )),
+
+    # NOTE: Demo performance depends on data
```

Comment:
There are other demos depending on data, for example `text_spotting_demo`. What is this note for?

Reply:
Special fix for you